runtime/src/bank.rs

```diff
@@ -4622,9 +4622,15 @@ impl Bank {
             };
 
             let mut compute_budget = ComputeBudget::default();
-            let prioritization_fee = compute_budget
-                .process_message(message, false, false, prioritization_fee_type_change)
+            let prioritization_fee_details = compute_budget
+                .process_instructions(
+                    message.program_instructions_iter(),
+                    false,
+                    false,
+                    prioritization_fee_type_change,
+                )
                 .unwrap_or_default();
+            let prioritization_fee = prioritization_fee_details.get_fee();
```

Comment:
cool, as long as this critical bit is the same fee as before the change (which looks the same), we shouldn't get divergent behavior

Reply:
Exactly, that's why `PrioritizationFeeDetails` passes through the deprecated fee without modification.